src/cli/onefuzz/azcopy.py

```diff
@@ -9,6 +9,17 @@ def find_azcopy() -> str:
     if azcopy:
         if not os.path.exists(azcopy):
             raise Exception(f"AZCOPY environment variable is invalid: {azcopy}")
+        elif os.path.isdir(azcopy):
+            contains_azcopy = "azcopy.exe" in [
+                f for f in os.listdir(azcopy) if os.path.isfile(os.path.join(azcopy, f))
+            ]
```

Comment:
Suggest simplifying this:
```suggestion
            contains_azcopy = os.path.isfile(os.path.join(azcopy, "azcopy.exe"))
```

Reply:
Actually, this should check for both `azcopy` and `azcopy.exe` to ensure that it works on Linux.

Reply:
(This may or may not need an `exists` check first…)